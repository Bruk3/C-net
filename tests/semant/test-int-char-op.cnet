int main() {
    int a = 1;
    char b = 2;

    return a + b;
}