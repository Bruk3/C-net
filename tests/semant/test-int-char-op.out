int main(int argc)
{
int a = (int : 1);
char b = (int : 2);
return (int : (int : a) + (char : b));
}

